programs/address-lookup-table/src/instruction.rs

```diff
@@ -1,12 +1,12 @@
 use {
-    crate::id,
     serde::{Deserialize, Serialize},
     solana_program::{
         clock::Slot,
         instruction::{AccountMeta, Instruction},
         pubkey::Pubkey,
         system_program,
     },
+    solana_sdk::address_lookup_table_program::id,
```

Comment:
```suggestion
    solana_program::address_lookup_table_program::id,
```
(plus you'll need to add solana-program as a dep in Cargo.toml)